tests/e2e/scripts/init.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Script to install pre-requisites on vsphere node to run e2e test
+
+PATH=$PATH:/usr/local/go/bin/:/usr/local/bin
+echo "Installing Kubectl"
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
+sudo mv kubectl /usr/local/bin/ && \
+chmod a+x /usr/local/bin/kubectl
+
+echo "Installing jq"
+sudo apt -y install jq
```

Comment:
Would it perhaps be better to run this script with `sudo` instead of having sudo in the script (something normally recommend to avoid).